foghorn_python/scripts/foghorn_python.py

```diff
@@ -0,0 +1,30 @@
+import sys, gzip, timeit, time, io, cStringIO, argparse
+from vcf_comp_funcs import * 
+param = sys.argv
+
+parser = argparse.ArgumentParser(prog="unzip_test_tool.py", formatter_class=argparse.RawDescriptionHelpFormatter,description="Command line python tool for transforming VCF files into cVCF files as required by the AMP-T2D GAIT analysis machines" +"\n" + "GT flag will compress genotypes splitting multialleic sites on to two lines." + "\n" + "The DS flag will compress dosage format. Currently not implemented for multi-allelics" + "\n" + "\n" + "Usage:" +"\n"+ "zcat inVCF.vcf.gz | python tool.py -o outfile -GT[DS]")
+parser.add_argument("-i","--vcf", help="Vcf file required. please pipe from zcat", type=argparse.FileType('r'), default=sys.stdin)
+parser.add_argument("-o","--out", help="Name of outfile",default="out")
```

Comment:
Use "required=True" to make sure this argument is supplied. Same for other applicable places.